package-lock.json

```diff
@@ -14,98 +14,270 @@
       }
     },
     "@babel/compat-data": {
-      "version": "7.11.0",
-      "resolved": "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.11.0.tgz",
-      "integrity": "sha512-TPSvJfv73ng0pfnEOh17bYMPQbI95+nGWc71Ss4vZdRBHTDqmM9Z8ZV4rYz8Ks7sfzc95n30k6ODIq5UGnXcYQ==",
-      "dev": true,
-      "requires": {
-        "browserslist": "^4.12.0",
-        "invariant": "^2.2.4",
-        "semver": "^5.5.0"
-      }
+      "version": "7.13.12",
+      "resolved": "https://team.skedify.io/npm/@babel%2fcompat-data/-/compat-data-7.13.12.tgz",
```

Comment:
Hmm, seems your installs are all getting resolved to the private npm repo. Considering this pkg is opensource, i'd revert this back to the regular registry. Ill push an updated lockfile